### ISSUE TITLE
Is this still in active development?

### ISSUE THREAD

**Issue by bobweston** [p/phpbb-tools#1]
Hi. I just stumbled upon this repo and was wondering if you are still working on it?

I've been using joomlatool's joomla-console application and it does a great job of letting me spin up new joomla sites from the command line.

I don't see a readme here. But I'm wondering if your intention is to create something similar for phpbb.

Thanks for any additional information you can provide.

**Comment by p** [comment]
Hi Bob, this repository has not been touched in some time and I no longer work on or with phpbb.

The scripts in this repository were intended for development and testing of phpbb itself. They will probably be of limited value to users of phpbb which is what I gather you are. QuickInstall (https://www.phpbb.com/customise/db/official_tool/phpbb3_quickinstall/) should be more appropriate for creating instances of phpbb.

You are of course welcome to use scripts here as you see fit.

**Comment by bobweston** [comment]
Hi Oleg,

Actually, I'm a developer and was looking for a way to fit phpbb into my
company's application deployment scheme. We use puppet for server
provisioning and all application deployment uses capistrano 3 and git. I
was hoping to find some way to automate the installation of plugins to
phpbb so we can do push-button deployment when deploying to various servers.

I

**Comment by p** [comment]
Right. I cannot think of any code I wrote which would accomplish this task, but in phpbb 3.1 you should be able to 1) add extension files to the board and 2) run enable method of phpbb extension manager (https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/phpbb/extension/manager.php#L244).
